roles/foreman_proxy_content/tasks/devel_install.yml

```diff
@@ -1,15 +1,18 @@
 ---
-- name: 'get existing rails server pid'
-  command: cat {{ base_foreman_directory }}tmp/pids/server.pid
-  register: existing_rails_server
-  become_user: vagrant
-  ignore_errors: True
+- name: 'Get Server Hostname'
+  shell: hostname -f
   delegate_to: "{{ foreman_proxy_content_server }}"
+  register: server_fqdn
+
+- name: 'Query Foreman server'
+  uri:
+    url: "{{ 'https://' ~ server_fqdn.stdout }}"
```

Comment:
I'm wondering if we should set this as a variable/fact and use it later on. You can even move it to `install.yaml` and share it. It would even be nicer if the URL wasn't guessed but actually retrieved from the `foreman_url` setting.

Reply:
I did another task that was doing the same thing to get the server name, so saving the value higher up sounds good.  I'll look into retrieving that url value.